### PR TITLE
add Update-WaykNow function

### PR DESCRIPTION
**Pull request by kacejot**
Update checks if WaykNow is already running and restores its condition after update is complete (if WaykNow UI was running, it will be running after update)

### CODE REVIEW

File: WaykNow/Public/WaykNowPackage.ps1 [hunk 1]
```diff
@@ -198,6 +198,21 @@ function Uninstall-WaykNow
 	}
 }
 
+function Update-WaykNow
```
Comment (awakecoding) [inline WaykNow/Public/WaykNowPackage.ps1]: can Update-WaykNow accept the -Force flag and pass it on to Install-WaykNow, to force the installation if the version is still the same?

File: WaykNow/Public/WaykNowProgram.ps1 [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
+	if (Get-IsWindows -Or $IsMacOS) {
+        $wayk_now_process = $(Get-Process | Where-Object -Property ProcessName -Like 'WaykNow')
+	} elseif ($IsLinux) {
+        $wayk_now_process = $(Get-Process | Where-Object -Property ProcessName -Like 'wayk-now')
+	}
+
+    return $wayk_now_process
+}
+
+function Get-NowService
```
Comment (awakecoding) [inline WaykNow/Public/WaykNowProgram.ps1]: Please rename to Get-WaykNowService

File: WaykNow/Public/WaykNowProgram.ps1 [hunk 3]
```diff
@@ -1,15 +1,40 @@
...
-            }
-        }
+    if (Get-IsWindows -And $PSEdition -Eq 'Desktop') {
+        $now_service = $(Get-Service | Where-Object -Property 'Name' -Like 'WaykNowService')
+	}
+
+    return $now_service
+}
+
+function Start-NowService
```
Comment (awakecoding) [inline WaykNow/Public/WaykNowProgram.ps1]: Please rename to Start-WaykNowService

File: WaykNow/Public/WaykNowPackage.ps1 (hunk 1 above)
Comment (kacejot) [reply WaykNow/Public/WaykNowPackage.ps1]: Of course. Will do

File: WaykNow/Public/WaykNowProgram.ps1 (hunk 2 above)
Comment (kacejot) [reply WaykNow/Public/WaykNowProgram.ps1]: Okay

File: WaykNow/Public/WaykNowProgram.ps1 (hunk 3 above)
Comment (kacejot) [reply WaykNow/Public/WaykNowProgram.ps1]: Okay

**Pull request merged**